.github/workflows/build.yml

```diff
@@ -66,7 +66,7 @@ jobs:
           cache: 'gradle'
 
       - name: Build with Gradle
-        run: ./gradlew build -x test -PjdkVersion=8
+        run: ./gradlew build -x test -PjdkVersion=8 -PskipDockerTests=false
```

Comment:
I'm a little confused, `-x test` is used here to skip tests, so why do we need to set skipDockerTests=false?

Reply:
yeah, it's redundant, I will remove it